migrations/src/test/scala/za/co/absa/enceladus/migrations/framework/fixture/UseCaseTestData.scala

```diff
@@ -89,34 +89,34 @@ class UseCaseTestData {
 
     db.insertDocument("schema",
       """{
-        |	"type": "struct",
-        |	"fields": [
-        |		{
-        |			"name": "field1",
-        |			"type": "string",
-        |			"nullable": true,
-        |			"metadata": {}
-        |		},
-        |		{
-        |			"name": "field2",
-        |			"type": "integer",
-        |			"nullable": true,
-        |			"metadata": {}
-        |		},
-        |		{
-        |			"name": "field3",
-        |			"type": "date",
-        |			"nullable": true,
-        |			"metadata": {
-        |				"pattern": "yyyy-MM-dd"
-        |			}
-        |		},
-        |		{
-        |			"name": "field4",
-        |			"type": "string",
-        |			"nullable": true,
-        |			"metadata": {}
-        |		}
+        | "type": "struct",
```

Comment:
Just a question. I'm wondering why the spacing is changed?

Reply:
It was tabbed, I changed it to spaces, since we use spaces everywhere. If you prefer I can switch it back.

Reply:
I prefer spaces as well.

Reply:
Yeah, we decided for spaces, so let's be consistent.